### PR TITLE
reference: refine expressions in characterset-and-collation.md

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? (Required)

refine expressions in characterset-and-collation.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: reference/sql/characterset-and-collation.md [hunk 1]
```diff
@@ -218,8 +218,8 @@ SELECT _utf8mb4'string' COLLATE utf8mb4_general_ci;
 
 规则如下：
 
-* 规则 1：如果指定了 `CHARACTER SET charset_name` 和 `COLLATE collation_name`，则直接使用 `CHARACTER SET charset_name` 和 `COLLATE collation_name`。
-* 规则 2：如果指定了 `CHARACTER SET charset_name` 且未指定 `COLLATE collation_name`，则使用 `CHARACTER SET charset_name` 和 `charset_name` 对应的默认排序规则。
+* 规则 1：如果指定了 `CHARACTER SET charset_name` 和 `COLLATE collation_name`，则直接使用 `charset_name` 和 `collation_name`。
```
Comment (wjhuang2016) [inline reference/sql/characterset-and-collation.md:221]: 则直接使用 `charset_name` 字符集和 `collation_name`排序规则。

Comment (yikeke) [inline reference/sql/characterset-and-collation.md:221]:
```suggestion
* 规则 1：如果指定了 `CHARACTER SET charset_name` 和 `COLLATE collation_name`，则直接使用 `charset_name`  字符集和 `collation_name` 排序规则。
```

File: reference/sql/characterset-and-collation.md [hunk 2]
```diff
@@ -268,8 +268,8 @@ SET collation_connection = @@collation_database;
 
 ## 字符集和排序规则的通用选择规则
 
-* 规则 1：如果指定了 `CHARACTER SET charset_name` 和 `COLLATE collation_name`，则直接使用 `CHARACTER SET charset_name` 和 `COLLATE collation_name`。
-* 规则 2：如果指定了 `CHARACTER SET charset_name` 且未指定 `COLLATE collation_name`，则使用 `CHARACTER SET charset_name` 和 `charset_name` 对应的默认排序规则。
+* 规则 1：如果指定了 `CHARACTER SET charset_name` 和 `COLLATE collation_name`，则直接使用 `charset_name` 和 `collation_name`。
```
Comment (yikeke) [inline reference/sql/characterset-and-collation.md:271]:
```suggestion
* 规则 1：如果指定了 `CHARACTER SET charset_name` 和 `COLLATE collation_name`，则直接使用 `charset_name` 字符集和 `collation_name` 排序规则。
```

File: reference/sql/characterset-and-collation.md (hunk 1 above)
Comment (yikeke) [reply reference/sql/characterset-and-collation.md]: Updated. PTAL again. @wjhuang2016

File: reference/sql/characterset-and-collation.md [hunk 3]
```diff
@@ -218,8 +218,8 @@ SELECT _utf8mb4'string' COLLATE utf8mb4_general_ci;
 
 规则如下：
 
-* 规则 1：如果指定了 `CHARACTER SET charset_name` 和 `COLLATE collation_name`，则直接使用 `CHARACTER SET charset_name` 和 `COLLATE collation_name`。
-* 规则 2：如果指定了 `CHARACTER SET charset_name` 且未指定 `COLLATE collation_name`，则使用 `CHARACTER SET charset_name` 和 `charset_name` 对应的默认排序规则。
+* 规则 1：如果指定了 `CHARACTER SET charset_name` 和 `COLLATE collation_name`，则直接使用 `charset_name`  字符集和 `collation_name` 排序规则。
+* 规则 2：如果指定了 `CHARACTER SET charset_name` 且未指定 `COLLATE collation_name`，则使用 `charset_name` 和 `charset_name` 对应的默认排序规则。
```
Comment (yikeke) [inline reference/sql/characterset-and-collation.md:222]:
```suggestion
* 规则 2：如果指定了 `CHARACTER SET charset_name` 且未指定 `COLLATE collation_name`，则使用 `charset_name` 字符集和 `charset_name` 对应的默认排序规则。
```

File: reference/sql/characterset-and-collation.md [hunk 4]
```diff
@@ -268,8 +268,8 @@ SET collation_connection = @@collation_database;
 
 ## 字符集和排序规则的通用选择规则
 
-* 规则 1：如果指定了 `CHARACTER SET charset_name` 和 `COLLATE collation_name`，则直接使用 `CHARACTER SET charset_name` 和 `COLLATE collation_name`。
-* 规则 2：如果指定了 `CHARACTER SET charset_name` 且未指定 `COLLATE collation_name`，则使用 `CHARACTER SET charset_name` 和 `charset_name` 对应的默认排序规则。
+* 规则 1：如果指定了 `CHARACTER SET charset_name` 和 `COLLATE collation_name`，则直接使用 `charset_name` 字符集和 `collation_name` 排序规则。
+* 规则 2：如果指定了 `CHARACTER SET charset_name` 且未指定 `COLLATE collation_name`，则使用 `charset_name` 和 `charset_name` 对应的默认排序规则。
```
Comment (yikeke) [inline reference/sql/characterset-and-collation.md:272]:
```suggestion
* 规则 2：如果指定了 `CHARACTER SET charset_name` 且未指定 `COLLATE collation_name`，则使用 `charset_name` 字符集和 `charset_name` 对应的默认排序规则。
```

**Pull request merged**